src/ext/java/org/opentripplanner/ext/transmodelapi/model/stop/QuayType.java

```diff
@@ -75,7 +86,7 @@ public static GraphQLObjectType create(
           )
           .dataFetcher(environment -> {
             String lang = environment.getArgument("lang");
```

Comment:
Is this fetcher using the correct language when the "language" argument is used (rather than "lang")?

Reply:
Yes I believe this is a feature of the GraphQLUtils.getLocale method. If "lang" is null, the getLocale will find the "language" argument and use that value.

Reply:
Ah you're right, the overloaded method doesn't do this. I will fix.

Reply:
I think this should work then:

```
Locale locale = lang != null ? GraphQLUtils.getLocale(environment, lang) : GraphQLUtils.getLocale(environment);
```